mdpow/tests/test_fep_script.py

```diff
@@ -0,0 +1,36 @@
+import tempdir as td
+import os
+import manifest
+from mdpow.equil import Simulation
+
+from mdpow.run import fep_simulation
+from mdpow.config import get_configuration
+
+class TestFEPScript(object):
+    
+    def setup(self):
+        self.tmpdir = td.TempDir()
```

Comment:
As we just checked: it *should* work with `tempdir.TempDir` when it's gc'ed. (If not then that's a bug in `tempdir`...)